### PR TITLE
Page AddItem

### LINKED ISSUE #35: 13.3 Como usuario quiero que la aplicación tenga un diseño de la vista AddItem para mobile y para PC
Como usuario, quiero que la página de agregar producto, de mi aplicación,  tenga un aspecto profesional y acogedor. Es importante que la vista sea adecuada cuando inicio sesión desde mi laptop o celular.

AC:

El usuario quiere poder ver la aplicación desde su celular y también desde su laptop sin afectarse el diseño de la aplicación

### PR DESCRIPTION
**Pull request by mailliwmogollon**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description
This PR adds the new layout to the Additem screen. 
Also added the React Icons library for application icons.

## Related Issue
Closes #35 
## Acceptance Criteria
The user wants to be able to see the application from his cell phone and also from his laptop without affecting the design of the application

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|   ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After

image
image

## Testing Steps / QA Criteria

From your terminal, pull down this branch with git pull additem-page and check that branch out with git checkout additem-page
npm install react-icons --save
npm start to launch the app
Fill in the "Shared token" field with the isa-collection token
Access the list screen

### CODE REVIEW

File: src/components/add-form/index.js [hunk 1]
```diff
@@ -3,48 +3,52 @@ import RadioInput from '../radio-input';
...
       <RadioInput
-        styles="time-frame-label"
         onChange={(event) => handleForm(event)}
         value="7"
         name="timeFrame"
         label="Soon"
+        className="time-frame-radio"
         required
         checked
       />
```
Comment (IsabellaSoares) [inline src/components/add-form/index.js:33]: What do you think about adding an `aria-label` to these radios? ❤️

**Review by IsabellaSoares — APPROVED** [review]
Thanks for all your work and effort, Mai! It was amazing!

File: src/pages/AddItem/index.js [hunk 2]
```diff
@@ -90,7 +90,11 @@ export const AddItemPage = () => {
         <ContentContainer>
           <form onSubmit={onSubmit} className="add-form">
             <AddForm handleForm={handleForm} formState={formState} />
-            <Button type="submit" disabled={formState.productName === ''}>
+            <Button
+              type="submit"
+              className="button-addform"
+              disabled={formState.productName === ''}
```
Comment (alxmcr) [inline src/pages/AddItem/index.js:96]: This line is so smart! Amazing ⭐ !

File: src/pages/AddItem/styles.css [hunk 3]
```diff
@@ -1,12 +1,42 @@
...
+  display: flex;
+  margin-bottom: 40px;
+  border-radius: 8px;
   width: 100%;
+  height: 40px;
+  font-size: 16px;
+  padding: 7px;
+}
+
+.button-addform:disabled {
```
Comment (alxmcr) [inline src/pages/AddItem/styles.css:29]: Good idea `:disabled` status! 🤩

File: src/components/add-form/index.js (hunk 1 above)
Comment (mailliwmogollon) [reply src/components/add-form/index.js:33]: Good idea

**Review by alxmcr — APPROVED** [review]
Approved ✔️ ! Congrats Mailliw!!! 🎊 🎉 💯  

I really appreciate your effort and energy for your work! Thank you so much! 🌟

File: src/components/add-form/index.js (hunk 1 above)
Comment (mailliwmogollon) [reply src/components/add-form/index.js]: Done

**Pull request merged**